Codecademy/gamut
# feat(Alert): wrap button under text + remove truncation on smaller screensizes

### Overview


Wraps the alert button below alert text  and does not truncate Alerts when on the smallest screensize. Also added font-smoothing to match Figma alerts 


<img width="600" alt="image" src="https://github.com/Codecademy/gamut/assets/58054751/abef95c6-6971-4d70-ab23-0f720acf8fe7">

### PR Checklist

- [x] Related to designs: [Designs](https://www.figma.com/file/ReGfRNillGABAj5SlITalN/%F0%9F%93%90-Gamut?type=design&node-id=30186-40358&mode=design&t=ljOUYqcXytX1bDlv-0)
- [x] Related to JIRA ticket: WEB-2520

#### Testing Instructions


1. Go to the Alert page
2. Alerts should looks identical to how they do now.
3.  Set your screensize to mobile
4. See the truncation be removed and main buttons wrapped to under the text 


#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/36837)  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/4412)  | [Portal Env](http://www.google.fr/ 'Named link title')   |
